common/utils/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -106,6 +106,20 @@ object SparkException {
       messageParameters = Map("message" -> msg),
       cause = cause)
   }
+
+  /**
+   * This is like the Scala require precondition, except it uses SparkException.
+   * @param requirement The requirement you want to check
+   * @param errorClass The error class to type if the requirement isn't passed
+   * @param messageParameters Message parameters to append to the message
+   */
+  def sparkRequire(requirement: Boolean,
```

Comment:
I doubt slightly about the name. The `spark` prefix doesn't bring any additional meaning. How about `checkArgs`, for instance. also cc @srielau

Reply:
Done!